### DISCUSSION TITLE
Another Way to to Caub (Not onc)

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#1019] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there one that I Can do in skiovox that isn't Onc Caub because it getting annoying every time it updates i have to redo EVERYTHING

**Comment by helloIlovegimkit** [comment]
Use caubflags

**Reply by CGRmags** [reply]
@helloIlovegimkit I cant do that SKioviox doesnt allows Chrome://flags i cant even go the website

**Reply by v4pl** [reply]
caub dns

**Comment by CGRmags** [comment]
@AshtonDavies Do you have a way?

**Reply by AshtonDavies** [reply, marked as answer]
Only other way of the three is [cauDNS](https://cauDNS.vercel.app).

**Reply by CGRmags** [reply]
does it block updates

**Reply by AshtonDavies** [reply]
> does it block updates
> 

I think so.

**Reply by CGRmags** [reply]
is there a way to to Caudns with multiple wifis

**Reply by AshtonDavies** [reply]
> is there a way to to Caudns with multiple wifis
> 

Set it on each network.

**Reply by helloIlovegimkit** [reply]
ye was going to say but I forgot what it was called
